app.py

```diff
@@ -437,19 +468,26 @@ def generate_dependency_graph(selected_libraries):
     [dash.dependencies.State("library-select", "value")],
 )
 def update_multi_options(search_value, value):
+    all_values = None
     if not search_value:
         raise PreventUpdate
-    res = requests.get("https://libraries.io/api/search?q={lib_name}&sort=stars".format(lib_name=search_value)).json()
-    new_libraries = [r["name"] for r in res]
-
-    exact_search = requests.get('https://libraries.io/api/search?q={}'.format(search_value)).json()
-    if search_value in [r["name"] for r in exact_search]:
-        new_libraries.append(search_value)
-
-    all_libraries = list(set(value + new_libraries))
-    all_libraries.sort(key=len)
-    all_values = [{"label": r, "value": r} for r in all_libraries]
-    return all_values
+    try:
+        if len(search_value) < 3:
```

Comment:
@murtazajafferji maybe the debounce thing will work better here.
But I have not used it and wanted to provide this so as to avoid bad user experience